src/InstallerPlugin.php

```diff
@@ -1,23 +1,22 @@
 <?php
 
 /**
- * Copyright MediaCT. All rights reserved.
- * https://www.mediact.nl
- * @deprecated
```

Comment:
It mentions it was deprecated. Any idea if this should get replaced or perhaps already is replaced with something else?